smithy-aws-traits/src/test/resources/software/amazon/smithy/aws/traits/errorfiles/tagging/invalid-tag-types.errors

```diff
@@ -0,0 +1,5 @@
+[WARNING] example.weather#Weather: Service marked `aws.api#TagEnabled` is missing an operation named 'ListTagsForResource.' | ServiceTagging
+[WARNING] example.weather#Weather: Service marked `aws.api#TagEnabled` is missing an operation named 'TagResource.' | ServiceTagging
+[WARNING] example.weather#Weather: Service marked `aws.api#TagEnabled` is missing an operation named 'UntagResource.' | ServiceTagging
+[ERROR] example.weather#City: Tag property must be a list shape targeting a member containing a pair of strings, or a Map shape targeting a string member. | TagResourcePropertyType
+[ERROR] example.weather#City: Resource does not have available tag CRUD operations. | TaggableResource
```

Comment:
This means there are no operations to create, read, update, or delete _tags_ not the resource itself. This happens because what ostensibly looks like one of the service-wide tagging operations is disqualified due to bad shape structure or missing members.

Reply:
So change to "Resource does not have CRUD operations for tagging" or "Resource does not have operations that perform CRUD on tags?"

Reply:
Something like: "Resource does not have tagging CRUD operations and is not compatible with service-wide tagging operations."